test/jasmine-test/spec/coreTests.js

```diff
@@ -305,6 +305,20 @@ var coreTests = [
         message: "should test for numeric zero in the context, prints the numeric zero"
       },
       {
+        name:     "negative integers are supported",
```

Comment:
`        name:     "inline params as integer",`

There are some tests that are similar to what these tests do; let's put your tests next to them.